pandas/core/indexes/accessors.py

```diff
@@ -157,6 +157,32 @@ class TimedeltaProperties(Properties):
     """
 
     def to_pytimedelta(self):
+        """
+        Return array of Timedeltas as `datetime.timedelta`.
+
+        Return an array of the Timedeltas in `datetime.timedelta` format with
+        the same length as the original Series.
+
+        Returns
+        -------
+        a : numpy.ndarray
+            1D array containing data with `datetime.timedelta` type.
+
+        Examples
+        --------
+        >>> s = pd.Series(pd.to_timedelta(np.arange(5), unit='d'))
```

Comment:
I *think* we need ``import datetime`` here

Reply:
I don't seem to need the import, but to clarify things, I've added more to the description.